esx_service/cli/vmdkops_admin.py

```diff
@@ -807,20 +807,23 @@ def tenant_ls_headers():
     headers = ['Uuid', 'Name', 'Description', 'Default_datastore', 'VM_list']
     return headers
 
-def generate_vm_list(vms_uuid):
-    """ Generate vm names with given list of vm uuid"""
-    # vms_uuid is a list of vm_uuid
-    # example: vms_uuid=["vm1_uuid", "vm2_uuid"]
-    # the return value is a string like this vm1,vm2
+def generate_vm_list(vm_list):
+    """ Generate vm names with given list of (vm_uuid, vm_name) from db"""
+    # vm_list is a list of (vm_uuid, vm_name) from db
+    # the return value is a comma separated string of VM names like this vm1,vm2
     res = ""
-    for vm_uuid in vms_uuid:
+    for vm_uuid, vm_db_name in vm_list:
```

Comment:
The idea is to not make the vm name in db the first choice, because it could be stale. Always try to resolve it from the host first, if couldn't resolve, then use one from db.